### ISSUE TITLE
Feature request: add .net generator

### ISSUE THREAD

**Issue by maxleiko** [orlandohill/waxeye#20]
Hello @orlandohill 

Could you take the time to map the Java generator for C# ? The syntax are quite the same.

**Comment by orlandohill** [comment]
Yes, I'd be happy to do that. It was always my intention to support C#.

**Comment by maxleiko** [comment]
:+1:

**Comment by bzaar** [comment]
Dear @orlandohill 

Let me start by saying that I've used Waxeye for a few years now in different projects (C and Java) and I simply love it. I've tried other parser generators (ANTLR) and they just didn't do it for me. Waxeye just worked, mostly like I expected, and did it well. Although it was the first time I saw any Racket code, I could see it was well written, clean code. And the documentation? Perfect. Nothing to add, nothing to take away. Thank you for making Waxeye and sharing it with the world.

I know you've been wanting this project to generate some revenue so how about this? I'm willing to pay to have the C# target added and I know you're the best person on the planet to do it. Please reach out if interested: https://t.me/bzaar

Best regards, 
Sergey Slepov

**Comment by orlandohill** [comment]
Hi Sergey,

Thanks for your kind words!

I'd be very happy to discuss development of a C# target for Waxeye.
